gulp-tasks/test.js

```diff
@@ -14,7 +14,7 @@ const runNodeTests = () => {
     options.push('--grep', global.cliOptions.grep);
   }
   return spawn(getNpmCmd(), ['run', 'test', '--',
-    `./test/${global.packageOrStar}/node/**/*.mjs`,
+    `./test/${global.packageOrStar}/node/**/*.{js,mjs}`,
```

Comment:
Why not write tests in `mjs`?

Reply:
- There's a large corpus of existing `workbox-build` tests written using CommonJS imports that I want to import as part of this PR to ensure that the modified build steps didn't break anything that worked in v2.
- The Node-targeted packages, like `workbox-build`, are sticking with CommonJS imports and exports internally. (We decided that we wanted the source to work with a modern version of Node without transpilation, which means no ESM, yet.) There's no need to write the tests as `.mjs` since they'll be consuming CommonJS modules.